src/schema/v1/fields/money.ts

```diff
@@ -53,26 +53,32 @@ export const amount = centsResolver => ({
   },
   resolve: (obj, options) => {
     const cents = centsResolver(obj)
-    if (typeof cents !== "number") {
-      return null
-    }
+    const symbol =
+      options.symbol || obj.symbol || symbolFromCurrencyCode(obj.currencyCode)
+    return moneyDisplay(cents, symbol, options)
+  },
+})
 
-    // Some objects return a currencyCode instead of a symbol.
-    const symbolFromCurrencyCode = obj.currencyCode
-      ? currencyCodes[obj.currencyCode.toLowerCase()] &&
-        currencyCodes[obj.currencyCode.toLowerCase()].symbol
-      : null
+export const symbolFromCurrencyCode = currencyCode => {
+  return currencyCode
+    ? currencyCodes[currencyCode.toLowerCase()] &&
+        currencyCodes[currencyCode.toLowerCase()].symbol
+    : null
+}
 
-    const symbol = options.symbol || obj.symbol || symbolFromCurrencyCode
+export const moneyDisplay = (cents, symbol, options) => {
```

Comment:
As stated above, if you pass in the symbol directly then you can remove this exported helper and rely on the existing resolver.